seeq/addons/correlation/__main__.py

```diff
@@ -5,73 +5,128 @@
 from getpass import getpass
 from urllib.parse import urlparse
 from seeq import sdk, spy
+# noinspection PyProtectedMember
+from seeq.spy._errors import *
+# noinspection PyProtectedMember
+from seeq.spy import _url
 from ._copy import copy
-from ._utils import sanitize_sdl_url, permissions_defaults
-from ._utils import get_user, get_user_group, get_datalab_project_id, addon_tool_management
+from .utils import get_user, get_user_group
 from . import correlation_udfs
 
 NB_EXTENSIONS = ['widgetsnbextension', 'plotlywidget', 'ipyvuetify', 'ipyvue']
 DEPLOYMENT_FOLDER = 'deployment'
 CORRELATION_NOTEBOOK = "correlation_analysis_master.ipynb"
 
 
-def install_app(sdl_url_, *, sort_key=None, permissions_group: list = None, permissions_users: list = None):
+def permissions_defaults(permissions_group: list, permissions_users: list):
+    if permissions_group is None:
+        permissions_group = ['Everyone']
+
+    if permissions_users is None:
+        permissions_users = []
+    return permissions_group, permissions_users
+
+
+def install_app(sdl_url_, *, sort_key='a', permissions_group: list = None, permissions_users: list = None):
     """
     Installs Correlation as an Add-on Tool in Seeq Workbench
 
     Parameters
     ----------
     sdl_url_: str
         URL of the SDL container.
-        E.g. https://my.seeq.com/data-lab/6AB49411-917E-44CC-BA19-5EE0F903100C/
-    sort_key: str, default None
+        E.g. `https://my.seeq.com/data-lab/6AB49411-917E-44CC-BA19-5EE0F903100C/`
+    sort_key: str, default 'a'
         A string, typically one character letter. The sort_key determines the
-        order in which the Add-on Tools are displayed in the tool panel
+        order in which the Add-on Tools are displayed in the tool panel.
     permissions_group: list
-        Names of the Seeq groups that will have access to each tool
+        Names of the Seeq groups that will have access to each tool. If None,
+        the "Everyone" group will be used by default.
     permissions_users: list
-        Names of Seeq users that will have access to each tool
+        Names of Seeq users that will have access to each tool. If None, no
+        individual users will be given access to the tool.
+
     Returns
     --------
     -: None
         Correlation Analysis will appear as Add-on Tool(s) in Seeq
         Workbench
     """
 
-    sdl_url_ = sanitize_sdl_url(sdl_url_)
-
-    if sort_key is None:
-        sort_key = 'a'
-
     permissions_group, permissions_users = permissions_defaults(permissions_group, permissions_users)
```

Comment:
Feels like given the name of the function, it should return the permission
defaults and shouldn't be settable.  As an alternative, what about just defining the 
default values as constants... and then removing the function and just setting like : 

```
permission_group = permission_group if permission_group else DEFAULT_GROUP
permission_users = permission_users if permission_users else DEFAULT_USERS
```

or: 

```
permission_group = permission_group or DEFAULT_GROUP
permission_users = permission_users or DEFAULT_USERS
```

Reply:
yes, I like that. Changed